stable/elasticsearch/templates/client-deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
 {{ toYaml .Values.client.podAnnotations | indent 8 }}
         {{- end }}
     spec:
+      serviceAccountName: {{ template "elasticsearch.serviceAccountName.client" . }}
```

Comment:
Why does this not use `template "elasticsearch.client.fullname"`?
Or alternatively why does the name of the client ServiceAccount not use this name?
Should they not be consistent?

Reply:
Oh I see what you are doing.